test/integration/vue.spec.ts

```diff
@@ -12,6 +12,16 @@ interface Error {
   rawMessage: string;
 }
 
+const printValue = (value: any) => {
+  if (typeof value === 'undefined') {
+    return 'undefined';
+  } else if (typeof value === 'object') {
+    return JSON.stringify(value);
+  } else {
+    return String(value);
+  }
+};
+
```

Comment:
I see this code duplicated in the next test - can we move it to the test helpers?

Reply:
It seems that we do not need to use these helper, jest has builtin```%p``` placeholder

Reply:
No more ```printValue``` in the code